app/config/parameters.yml.dist

```diff
@@ -13,6 +13,13 @@ parameters:
           username: root
           password: secret
           table: storage
+          encryption:
+            type: mcrypt
+            options:
+                cipher:
+                mode:
+                key:
+                iv:
```

Comment:
This is a default config specifying the options. I wasn't sure if there should be default values in there